### PR TITLE
geodesic_prompt(): add PROMPT_STYLE handler

### LINKED ISSUE #150: Add Support for Custom Prompts
## what
* Add support for `PROMPT_STYLE` environment variable that alters the behavior of the shell prompt

## why
* The current utf8 heavy prompt breaks some terminals
* Users have mentioned they'd prefer a plain prompt

### PR DESCRIPTION
**Pull request by tamsky**
## what
* Add support for a "plain" prompt 

## why
* UTF8 glyphs break some peoples terminals

Fixes #150

### CODE REVIEW

File: rootfs/etc/profile.d/prompt.sh [hunk 1]
```diff
@@ -28,12 +28,25 @@ function terraform_prompt() {
...
+          CROSS_MARK=$'\u274C '                 # '❌'
+          ;;
+  esac
 
-  WHITE_HEAVY_CHECK_MARK=$'\u2705 '
-  BLACK_RIGHTWARDS_ARROWHEAD=$'\u27A4 '
-  TWO_JOINED_SQUARES=$'\u29C9 '
-  CROSS_MARK=$'\u274C '
-
+  # why do we 'export' STATUS?
```
Comment (osterman) [inline rootfs/etc/profile.d/prompt.sh]: This might be legacy. Don't think the `export` is needed.

Comment (osterman) [reply rootfs/etc/profile.d/prompt.sh]: if it works without it, drop it.

File: rootfs/etc/profile.d/prompt.sh [hunk 2]
```diff
@@ -28,12 +28,25 @@ function terraform_prompt() {
 # Define our own prompt
 PROMPT_HOOKS+=("geodesic_prompt")
 function geodesic_prompt() {
+       
+  case $PROMPT_STYLE in
+      plain)
+          # 8859-1 codepoints:
+          WHITE_HEAVY_CHECK_MARK=$(tput bold)$(tput setab 2)$'X'$(tput sgr0)' '
+          BLACK_RIGHTWARDS_ARROWHEAD=$'=> '
+          TWO_JOINED_SQUARES=$'¤ '  # perhaps § instead?
```
Comment (osterman) [inline rootfs/etc/profile.d/prompt.sh]: Yea, I like `§`

Comment (osterman) [reply rootfs/etc/profile.d/prompt.sh]: perhaps we should generalize the names? e.g. it's no longer "Two joined names"

File: rootfs/etc/profile.d/prompt.sh [hunk 3]
```diff
@@ -28,12 +28,25 @@ function terraform_prompt() {
 # Define our own prompt
 PROMPT_HOOKS+=("geodesic_prompt")
 function geodesic_prompt() {
+       
+  case $PROMPT_STYLE in
```
Comment (osterman) [inline rootfs/etc/profile.d/prompt.sh]: Perfect! thanks for making it a `case` statement.

File: rootfs/etc/profile.d/prompt.sh (hunk 2 above)
Comment (tamsky) [reply rootfs/etc/profile.d/prompt.sh]: +1

I'll try my hand at some more generic sounding names?

File: rootfs/etc/profile.d/prompt.sh (hunk 1 above)
Comment (tamsky) [reply rootfs/etc/profile.d/prompt.sh]: dropped.

**Comment by tamsky** [comment]
example rendering:
image

**Pull request merged**